app/models.py

```diff
@@ -352,6 +352,21 @@ class OrganisationTypes(db.Model):
     annual_free_sms_fragment_limit = db.Column(db.BigInteger, nullable=False)
 
 
+class OrganisationPermission(db.Model):
+    __tablename__ = "organisation_permissions"
+    id = db.Column(UUID(as_uuid=True), primary_key=True, nullable=False, default=uuid.uuid4)
+
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
+    organisation = db.relationship("Organisation")
```

Comment:
It would be nice to have a backref on this so that we can access `Organisation.permissions` directly (we do similar things for Service/ServicePermission)

```suggestion
    organisation = db.relationship("Organisation", backref="permissions")
```